eZ/Publish/Core/IO/FilePathNormalizerInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\IO;
+
+interface FilePathNormalizerInterface
```

Comment:
Anything declared outside of API/SPI namespaces needs to be internal
```suggestion
/**
 * @internal
 */
interface FilePathNormalizerInterface
```